style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: Inter, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+  background: #e5e5e5;
+}
+
+#skeleton {
+  width: 375px;
+  margin: 0 auto;
+  background: #fff;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 24px;
+  margin-right: 28px;
+  padding-top: 27px;
+}
+
+p#welcome {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+#menu {
+  width: 16px;
+  height: 10px;
+}
+
+#page {
+  background-image: url("/images/skeleton.svg");
+  background-repeat: no-repeat;
+}
+
+#intro {
+  margin: 0 24px;
+  padding-top: 74px;
+  padding-bottom: 25px;
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+#denis {
+  color: #172b4d;
+}
+
+#trade {
+  color: #36b37e;
+}
+
+#advert {
+  margin: 0 24px;
+  padding-bottom: 60px;
+}
```

Comment:
- [ ] Kindly make sure to follow the style, font according to the Figma design for your headings and paragraph.